src/System.ServiceModel.Http/tests/ServiceModel/SecurityBindingElementTest.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Linq;
+using System.ServiceModel;
+using System.ServiceModel.Channels;
+using Xunit;
+
+public static class SecurityBindingElementTest
+{
+    [Fact]
+    public static void Create_HttpBinding_SecurityMode_TransportWithMessageCredential()
+    {
+        BasicHttpBinding binding = new BasicHttpBinding(BasicHttpSecurityMode.TransportWithMessageCredential);
```

Comment:
Nit: this would make a great [Theory] based test, since they seem to be identical except for the enum value.